token-client/src/test/java/com/sap/cloud/security/xsuaa/client/DefaultOAuth2TokenKeyServiceTest.java

```diff
@@ -118,6 +123,22 @@ public void retrieveTokenKeys_errorOccurs_throwsServiceException() throws IOExce
 				.hasMessageContaining(errorMessage);
 	}
 
+	@Test
+	public void retrieveTokenKeys_app2Service_ProofToken() throws IOException {
+		CloseableHttpResponse response = HttpClientTestFactory.createHttpResponseWithHeaders(jsonWebKeysAsString,
+				new BasicHeader[] { new BasicHeader(X_OSB_PLAN, "plan") });
```

Comment:
Let's try to get a better example from colleagues with the actual comma-separated list format